html-forms/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Coding Form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
+      <form action="" method="get" class="coding-form">
+        <div>
+          <label for="email">Email</label>
+          <div>
+            <input type="text" name="email" id="email" required/>
+          </div>
+        </div>
+      </form>
+      <form action="" method="get" class="coding-form">
+        <div>
+          <label for="options">Why is coding cool?</label>
+          <div>
+            <select name="options" id="options">
+              <option value="Select an option">Select an option</option>
+              <option value="Because it's interesting">Because it's interesting</option>
+              <option value="Because it's fun">Becuase it's fun</option>
+              <option value="Because it's powerful">Because it's powerful</option>
+            </select>
+          </div>
+        </div>
+      </form>
+      <form action="" method="get" class="coding-form">
+        <p>
+          Favorite Language (choose one)
+        </p>
+        <div>
+          <input type="radio" name="language" value="HTML"/> HTML
```

Comment:
you need a label for each of the inputs. I recommend wrapping the label around the input.